### DISCUSSION TITLE
Fixieren von Widgets

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#20905] [Q&A]
Die Widgets für 3D und Schnellaktionen werden von mir leider immer wieder auf dem Bildschirm verschoben. Kann man die auch fixieren? Das ist ziemlich nervig do

**Comment by DmitryAlexei** [comment]
What version of the app do you use?
When does the problem occur? When you tap on the button or when you change map orientation or ...?
Could you please provide short video of your issue?

**Comment by ghost** [comment]
Osmand+ 4.8.6 auf Android 14
Wenn ich in der Karte zoome, drehe oder eine neue Route erstelle und dabei versehentlich auf einen dieser Button komme. Dann verschieben sie sich tlw sogar bis in die untere Taskleiste von Android. Da ich sie zudem transparent habe, ist es manchmal ein Suchspiel, sie wieder zu finden.
